### ISSUE TITLE
ObMasterLayout top navigation  parent menu item without URL 

### ISSUE THREAD

**Issue by BenjaminHofstetter** [oblique-bit/oblique#130]
### Description

```true
I have an Oblique MasterLayout with a top navigation menu like this. 


 topNavigation: ObINavigationLink[] = [
    {
      label: 'topNavigation.overview',
      url: 'overview',
    },
    {
      label: "topNavigation.food_and_nutrition",
      url: "food_and_nutrition",
      children: [
        {
          label: "topNavigation.meat_inspection_statistics",
          url: 'meat-inspection-statistics',
        },
      ]
    },
    {
      label: "topNavigation.animals",
      url: 'animals',
    },
    {
      label: "topNavigation.animal_disease_notifications",
      url: 'animal-disease-notifications',
      children: [
        {
          label: "topNavigation.animal_disease_notifications_bulletin",
          url: 'bulletin',
        },
        {
          label: "topNavigation.animal_disease_notifications_evaluation",
          url: 'evaluation',
          children: [
            {
              label: "topNavigation.animal_disease_notifications_evaluation_notification",
              url: 'notification',
            },
            {
              label: "topNavigation.animal_disease_notifications_evaluation_animal",
              url: 'animal',
            },
            {
              label: "topNavigation.animal_disease_notifications_evaluation_pest",
              url: 'pest',
            },
          ]
        },
        {
          label: "topNavigation.animal_disease_notifications_about",
          url: 'about',
        }
      ]
    }
  ];



My question is related to this section of the menu.

```typescript
 {
          label: "topNavigation.animal_disease_notifications_evaluation",
          url: 'evaluation',
          children: [
            {
              label: "topNavigation.animal_disease_notifications_evaluation_notification",
              url: 'notification',
            },
            {
              label: "topNavigation.animal_disease_notifications_evaluation_animal",
              url: 'animal',
            },
            {
              label: "topNavigation.animal_disease_notifications_evaluation_pest",
              url: 'pest',
            },
          ]
        },
```

What i want is a parent menu entry (label: "topNavigation.animal_disease_notifications_evaluation") without a route / component assigned to it. I am only interested in the routes of the sub menu. 

Is this possible?
```


### Use case

_No response_

### Relation to existing issue

_No response_

### Proposed solution

_No response_

**Comment by nina-egger** [comment]
Hello @BenjaminHofstetter 

I tested this in the sandbox and it doesn't seem to be possible even if you leave url empty. I will open a ticket to discuss of this is actually a bug or a feature request.

**Comment by nina-egger** [comment]
@gillerr can you take a look at this?

IMO this should be a bug. If the URL of a Nav Link (of a child or not) is empty we should not try to navigate at all. Can I declare this as a bug or is it a feature request instead?

**Comment by gillerr** [comment]
This is a feature request, not a bug. It has been decided that each level that has children still is a page (like Confluence works)  Which means that an url has to be provided for each level. 

We could however cover both use cases. If a URL is provided, then we have our current solution, when not, then the link is replaced by a button and simply opens the next level. I will create the relevant tasks in our backlog.

As a side note, I'm not a fan of multi level navigation and I don't know why it has been implemented, it's not even part of the Web Guidelines created by the Federal Chancellery.

**Comment by gillerr** [comment]
This has been fixed with the version 12.1.0
